libs/application-generic/src/usecases/upsert-preferences/upsert-preferences.usecase.ts

```diff
@@ -70,18 +104,11 @@ export class UpsertPreferences {
       return this.deletePreferences(command, foundId);
     }
 
-    const builtPreferences = buildWorkflowPreferences(command.preferences);
-
-    const builtCommand = {
-      ...command,
-      preferences: builtPreferences,
-    };
-
```

Comment:
We no longer build a full preference set for all Preference types, ensuring that `PATCH` operations mutate only the minimal preference channels.